doc/ref/configuration/minion.rst

```diff
@@ -1181,6 +1181,17 @@ The password used for HTTP proxy access.
 
     proxy_password: obolus
 
+``no_proxy``
+------------------
+
+Default: ``[]``
+
+List of hosts to bypass proxy
+
+.. code-block:: yaml
+
+    no_proxy: [ '127.0.0.1', 'foo.tld' ]
```

Comment:
IMHO, `bypass_proxy` or `exclude_proxy` might be better variable names here.

Reply:
Just my 2c - no_proxy is widely used as the name of the environment variable on Linux and a bunch of others.  So, it's purpose and syntax are well understood.

Reply:
Thats why i selected no_proxy just because of the defacto relationship